int32 servo_id
float64 angle    # [rad]
---
bool success